bibtexparser/bibdatabase.py

```diff
@@ -66,6 +67,23 @@ def __init__(self):
         #: List of fields that should not be updated when resolving crossrefs
         self._not_updated_by_crossref = ['_FROM_CROSSREF']
 
+    def __eq__(self, other: Any) -> bool:
+        """Equality == method."""
+        if (
+            isinstance(other, BibDatabase)
+            and other.entries == self.entries
+            and other.comments == self.comments
+            and other.strings == self.strings
+            and other.preambles == self.preambles
```

Comment:
I know the `self._not_updated_by_crossref` is quasi-static, but as it is not, I think I would like to have it included in the equation.

Reply:
Done, I feel like the structure around this crossref isn't very solid. Should we also include `self._crossref_updated`?

Reply:
Sure, all attributes ;-) Thanks!